Documentation/Testing/PreviewTemplates.md

```diff
@@ -2,19 +2,4 @@
 
 # Testing Preview Workbook Templates
 
-You can test templates that are still work in progress or simply not ready to be exposed to all users. To do this you need to add the property `"isPreview: true"` in settings.json.

```

Comment:
should we just remove this folder?

Reply:
we could but i didn't want to remove it if anything had linked to it from anywhere else.